tests/tests/p2p_node_tests.cpp

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2017 Bitshares Foundation, and contributors.
```

Comment:
2019?

Reply:
> 2019?

That's when I started building this test. Hehe.